src/smart-components/workflow/add-stages/set-stages.js

```diff
@@ -42,35 +48,33 @@ const SetStages = ({ formData, handleChange, options, title }) => {
     handleChange({ wfGroups: values });
   };
 
+  const loadGroupOptions = (inputValue) => fetchFilterGroups(inputValue);
```

Comment:
The async callback works and the options are populated correctly. But i would like to see this function debounced. Right now you call this function on every key stroke which might not work very well on slower networks. Promises can be executed in wrong order.

Example is here: https://github.com/ManageIQ/catalog-ui/blob/master/src/forms/portfolio-form.schema.js#L39